app/viewmodels/checkAnswers/BusinessAddressWithoutIDSummary.scala

```diff
@@ -31,29 +31,15 @@ object BusinessAddressWithoutIDSummary {
     answers.get(NonUKBusinessAddressWithoutIDPage).map {
       answer =>
         SummaryListRowViewModel(
-          key = "businessWithoutIDAddress.checkYourAnswersLabel",
+          key = s"$NonUKBusinessAddressWithoutIDPage.business.checkYourAnswersLabel",
           value = formatAddress(answer),
           actions = Seq(
-            ActionItemViewModel("site.change", routes.NonUKBusinessAddressWithoutIDController.onPageLoad(CheckMode).url)
-              .withVisuallyHiddenText(messages("businessWithoutIDAddress.change.hidden"))
+            changeAction(
+              NonUKBusinessAddressWithoutIDPage.toString,
```

Comment:
only summary file using the page.toString - don't see any issues though

Reply:
Thanks. I forgot to update the other ones.